ado_asana_sync/sync/sync.py

```diff
@@ -158,6 +159,27 @@ def is_current(self, a: App) -> bool:
         return True
 
 
+def safe_get(obj, *attrs_keys):
+    """
+    Safely retrieves nested attributes from an object.
+
+    Args:
+        obj: The object to retrieve attributes from.
+        *attrs_keys: Variable number of attribute keys.
+
+    Returns:
+        The value of the nested attribute if found, else None.
+    """
+    for attr_key in attrs_keys:
+        if isinstance(obj, dict):
+            obj = obj.get(attr_key)
+        else:
+            obj = getattr(obj, attr_key, None)
+        if obj is None:
+            return None
+    return obj
```

Comment:
**Suggestion:** The `safe_get` function could be improved by using a try/except block instead of checking if the object is None in each iteration. This would make the function more pythonic and efficient.
```suggestion
def safe_get(obj, *attrs_keys):
    try:
        for attr_key in attrs_keys:
            obj = obj.get(attr_key) if isinstance(obj, dict) else getattr(obj, attr_key)
        return obj
    except AttributeError:
        return None
```